test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -132,19 +132,18 @@ Describe "Send-MailMessage DRT Unit Tests" -Tags CI, RequireSudoOnUnix {
         $mail = Read-Mail
 
         $mail.FromAddress | Should -BeExactly $InputObject.From
+        $mail.ToAddresses.Count | Should -BeExactly ($InputObject.To.Count + $InputObject.Cc.Count + $InputObject.Bcc.Count)
         $mail.ToAddresses | Should -BeIn ([array]$InputObject.To + $InputObject.Cc + $InputObject.Bcc)
 
         $mail.Headers["From"] | Should -BeExactly $InputObject.From
+        $mail.Headers["To"] | Should -Not -BeNullOrEmpty
         $mail.Headers["To"].Split(", ") | Should -BeExactly $InputObject.To
-        If($mail.Headers["Cc"])
+        If($InputObject.Cc)
         {
+            $mail.Headers["Cc"] | Should -Not -BeNullOrEmpty
             $mail.Headers["Cc"].Split(", ") | Should -BeExactly $InputObject.Cc
         }
-        If($mail.Headers["Bcc"])
```

Comment:
Please add comment for the case that we can not test the case.

Reply:
Fixed! Thank you very much @iSazonov! Sorry for the delay.